python/cuml/cuml_accel_tests/test_set_output.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.datasets import make_blobs
+
+from cuml.accel.core import ACCELERATED_MODULES
+from cuml.accel.estimator_proxy import ProxyBase
+
+
+def proxy_classes_with_set_output():
+    for mod in ACCELERATED_MODULES:
+        importlib.import_module(mod)
+
+    return sorted(
+        (
+            cls
+            for cls in ProxyBase.__subclasses__()
+            if hasattr(cls, "set_output")
+        ),
+        key=lambda cls: cls.__name__,
+    )
+
+
+@pytest.mark.parametrize("cls", proxy_classes_with_set_output())
+def test_set_output(cls):
```

Comment:
This test checks _all_ our estimators that support `set_output`. The test is less thorough than that in `test_estimator_proxy`, but sufficiently checks that everything is plumbed together properly for each class. These tests combined provide adequate test coverage IMO.